CONTRIBUTING.md

```diff
@@ -134,7 +137,7 @@ make check-pypi-packaging
 ### Requirements for Pull Requests (PR)
 
 - Use `pre-commit` (see [below](#checkerslintersformatters--pre-commit)).
-- Use common sense when creating commits, not too big, not too small. You can also squash them at the end of review. See [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/).
+- Use common sense when creating commits, not too big, not too small. You can also squash them at the end of review.
```

Comment:
The blog post can help I think. I would leave the link there.

Reply:
OK, I see you move that part.

I like that part, but don't we duplicate a lot of things?